ImGuiFileDialog.cpp

```diff
@@ -75,300 +103,330 @@ ImGuiFileDialog::~ImGuiFileDialog()
 }
 
 /* Alphabetical sorting */
-static int alphaSort(
-	const void *a, const void *b)
+static int alphaSort(const struct dirent **a, const struct dirent **b)
 {
-	return strcoll(((dirent*)a)->d_name, ((dirent*)b)->d_name);
+    return strcoll((*a)->d_name, (*b)->d_name);
 }
 
-static bool stringComparator(FileInfoStruct a, FileInfoStruct b)
```

Comment:
Not used anywhere so removed for cleanness. Feel free to leave it if you like - it's your code.